### ISSUE TITLE
Design corygalynaCORE GUI Component

### ISSUE THREAD

**Issue by corygalyna** [ZORALab/demo-web-prestige#3]
# Description
```
Please provide a short description of what feature you're looking forward to
add into the existing repository below. Please include the story behind your
idea as well to give a better visualization of your idea.
```
The design has to be prestige so a new set of GUI component is required. Let's called it `prestigeCORE` for now.

# Expected Behavior
```
Please specify the expected behavior of your requested feature. Some great and
helpful pointers are your expected interface (e.g. command patterns, simple
sketches of the user interface, etc).
```
`prestigeCORE` GUI component is developed for this demo.

# Current Absent Behavior
```
Please specify the current behavior (e.g. workaround, blockage, etc).
```
No such component. Clean slate.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Hey, I'm gonna name it `corygalynaCORE` then. A present for you in the mean time.

**Comment by corygalyna** [comment]
Oh then I'll make sure you're going to make it tip-top shape. =) You asked for it.

**Comment by hollowaykeanho** [comment]
Sure. Germany style?

**Comment by corygalyna** [comment]
100%... add a little Ukrainian flavour please. TQ.

**Comment by hollowaykeanho** [comment]
Done.

**Comment by hollowaykeanho** [comment]
Sorry.. marked the wrong ticket.

**Comment by hollowaykeanho** [comment]
favourite color?

**Comment by corygalyna** [comment]
https://www.colorhexa.com/87ceeb

**Comment by hollowaykeanho** [comment]
That's tough. Let me see how can I workaround it.

**Comment by corygalyna** [comment]
Still working on this? Are you sure you can meet the deadline?

**Comment by hollowaykeanho** [comment]
I'm working with item #8 as well. Co-dev this component against it.

**Comment by hollowaykeanho** [comment]
Implemented in 307365d322048e9207e38b7932534e73d3dc9d03
